doc/samtools-mpileup.1

```diff
@@ -482,7 +483,43 @@ The
 .B -E
 option can be used to make it ignore the contents of the BQ:Z tag and
 force it to recalculate the BAQ scores by making a new alignment.
-
+.PP
+.SH EXAMPLES
+.PP
+Using range:
+.PP
+with implicit index files in1.bam.<ext> and in2.sam.gz.<ext>,
+.IP
+.B samtools mpileup in1.bam in2.sam.gz -r <name>:<range>
+.PP
+with explicit index files,
+.IP
+.B samtools mpileup in1.bam in2.sam.gz <in1 index path> <in2 index path> -X -r <name>:<range>
+.PP
+with fofn having input file names, with implicit index files present along with inputs,
```

Comment:
Nit-picky: maybe be more explicit with "fofn" by adding "file of", eg "with fofn being an file of input filenames", just to reinforce what the acronym stands for.

Reply:
done.